examples/preprocessing/plot_scaling_importance.py

```diff
@@ -137,3 +180,42 @@
 plt.tight_layout()
 
 plt.show()
+
+# %%
+# Effect of rescaling on model's performance
+# =======================================
+#
+# Here we show prediction accuracies in scaled and unscaled data.
+
+from sklearn.metrics import accuracy_score
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import LogisticRegression
+
```

Comment:
I quickly try with the following values:

```python
Cs = np.logspace(-5, 5, 20)
```

This is interesting because the results do not change so much:

```
Prediction accuracy for the normal test dataset with PCA
70.37%

Prediction accuracy for the standardized test dataset with PCA
96.30%
```

But it is interesting to look at the `C_` chosen:

```
unscaled_clf[-1].C_
array([0.00037927, 0.00037927, 0.00037927])
scaled_clf[-1].C_
array([20.69138081, 20.69138081, 20.69138081])
```